.github/workflows/linux_release.yml

```diff
@@ -74,6 +73,15 @@ jobs:
       shell: bash
       run: cmake --build . --target package -j 4
 
+    - name: Upload Tarball as artifact for testing
+      uses: actions/upload-artifact@v4
+      with:
+        name: energyplus-${{ matrix.os }}
+        path: build/EnergyPlus-*-x86_64.tar.gz
+        if-no-files-found: error
+        retention-days: 7
+        overwrite: false
+
```

Comment:
On all three platforms, I now upload the tarball/zip as an artifact in the first job, and then test the artifact in a fresh container in a second job.